test/unit/init/github_test.js

```diff
@@ -10,95 +10,86 @@
 'use strict';
```

Comment:
This file is more or less completely redone, but I can't find out how to tell Github to stop diff failing. You'll be better served clicking "View" above to review.